protocols/v2/roles-logic-sv2/src/errors.rs

```diff
@@ -142,6 +146,7 @@ impl Display for Error {
             JobNotUpdated(ds_job_id, us_job_id) => write!(f, "Channel Factory did not update job: Downstream job id = {}, Upstream job id = {}", ds_job_id, us_job_id),
             TargetError(e) => write!(f, "Impossible to get Target: {:?}", e),
             HashrateError(e) => write!(f, "Impossible to get Hashrate: {:?}", e),
+            LogicErrorMessage(e) => write!(f, "Message is well formateed but can not be handled: {:?}", e),
```

Comment:
Simple typo here. "Message is well `formatted` but ...."

Reply:
thanks